app/views/shared/_no_pii_banner.html.erb

```diff
@@ -1,3 +1,3 @@
-<div class='padding-y-1 bg-secondary-darker text-white fs-12p line-height-1 center'>
+<div class='h1 padding-y-4 bg-secondary-darker text-white fs-12p line-height-1 center'>
```

Comment:
`h1` is [a BassCSS class](https://github.com/basscss/basscss-sass/blob/v3.0.0/_type-scale.scss#L68). A [design system class](https://designsystem.digital.gov/utilities/font-size-and-family/) like `font-sans-lg` would have been preferable toward moving off BassCSS.

```suggestion
<div class='font-sans-lg padding-y-4 bg-secondary-darker text-white fs-12p line-height-1 center'>
```